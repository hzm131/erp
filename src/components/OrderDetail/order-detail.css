.table-color td{
    border:1px solid #DEDEDE
}
