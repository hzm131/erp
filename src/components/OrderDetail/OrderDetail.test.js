import React from 'react'
import { mount } from 'enzyme'
import OrderDetail from './OrderDetail'

it('should render using mount', ()=>{
  const mounted = mount(<OrderDetail />)

  expect(mounted).toMatchSnapshot()
})
