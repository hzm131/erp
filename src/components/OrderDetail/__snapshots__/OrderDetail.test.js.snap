// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render using mount 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#F5F5F5",
      "borderRadius": "8px",
      "flex": 1,
      "fontSize": "12px",
      "padding": "12px",
    }
  }
>
  <div>
    <div
      style={
        Object {
          "display": "flex",
          "justifyContent": "space-between",
        }
      }
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <span>
          姓名:
        </span>
        <span>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
                "width": "120px",
              }
            }
            type="text"
            value=""
          />
        </span>
      </div>
      <div>
        <div
          className="ant-radio-group ant-radio-group-outline"
        >
          <label
            className="ant-radio-button-wrapper"
          >
            <span
              className="ant-radio-button"
              style={Object {}}
            >
              <input
                checked={false}
                className="ant-radio-button-input"
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value={0}
              />
              <span
                className="ant-radio-button-inner"
              />
            </span>
            <span>
              男
            </span>
          </label>
          <label
            className="ant-radio-button-wrapper"
          >
            <span
              className="ant-radio-button"
              style={Object {}}
            >
              <input
                checked={false}
                className="ant-radio-button-input"
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value={1}
              />
              <span
                className="ant-radio-button-inner"
              />
            </span>
            <span>
              女
            </span>
          </label>
        </div>
      </div>
      <div>
        <span>
          购买日期:
        </span>
        <span>
          <span
            className="ant-calendar-picker"
            onBlur={[Function]}
            onFocus={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
              }
            }
          >
            <div
              key="trigger"
              onKeyDown={[Function]}
            >
              <input
                className="ant-calendar-picker-input ant-input ant-input-disabled"
                disabled={true}
                placeholder="Select date"
                readOnly={true}
                value="2018-12-17"
              />
              <span
                className="ant-calendar-picker-icon"
              />
            </div>
          </span>
        </span>
      </div>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "justifyContent": "space-between",
          "marginTop": "10px",
        }
      }
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <span>
          电话:
        </span>
        <span>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
              }
            }
            type="text"
            value=""
          />
        </span>
      </div>
      <div>
        <span>
          年龄:
        </span>
        <span>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
                "width": "40px",
              }
            }
            type="text"
            value=""
          />
        </span>
      </div>
      <div>
        <span>
          职业:
        </span>
        <span>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
                "width": "120px",
              }
            }
            type="text"
            value=""
          />
        </span>
      </div>
    </div>
    <div
      style={
        Object {
          "display": "flex",
          "justifyContent": "space-between",
          "marginTop": "10px",
        }
      }
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <span>
          卡号:
        </span>
        <span>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "12px",
              }
            }
            type="text"
            value=""
          />
        </span>
      </div>
      <div>
        <span>
          生日:
        </span>
        <span
          style={
            Object {
              "marginRight": "12px",
            }
          }
        >
          <span
            className="ant-calendar-picker"
            onBlur={[Function]}
            onFocus={[Function]}
            style={
              Object {
                "fontSize": "12px",
                "marginLeft": "10px",
              }
            }
          >
            <div
              key="trigger"
              onClick={[Function]}
              onKeyDown={[Function]}
              onMouseDown={[Function]}
              onTouchStart={[Function]}
            >
              <input
                className="ant-calendar-picker-input ant-input"
                placeholder="Select date"
                readOnly={true}
                value=""
              />
              <span
                className="ant-calendar-picker-icon"
              />
            </div>
          </span>
        </span>
        <span>
          <button
            className="ant-btn ant-btn-circle ant-btn-sm ant-btn-icon-only"
            onClick={[Function]}
            style={
              Object {
                "color": "#CDCDCD",
              }
            }
            type="button"
          >
            <i
              className="anticon anticon-close"
            />
          </button>
        </span>
      </div>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#DEDEDE",
          "height": "2px",
          "marginTop": "15px",
          "width": "100%",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "marginTop": "15px",
        }
      }
    >
      <span>
        <span
          checked={false}
          className="ant-switch"
          defaultChecked={false}
          onChange={[Function]}
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseUp={[Function]}
          tabIndex={0}
        >
          <span
            className="ant-switch-inner"
          />
        </span>
      </span>
      <span
        style={
          Object {
            "marginLeft": "15px",
          }
        }
      >
        无配镜处方
      </span>
    </div>
  </div>
  <div
    className=""
    style={
      Object {
        "marginTop": "20px",
        "width": "650px",
      }
    }
  >
    <table
      key="0"
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": "10px",
          "marginTop": "2px",
          "width": "650px",
        }
      }
    >
      <tbody>
        <tr>
          <td
            colSpan={7}
            style={
              Object {
                "padding": "8px",
                "textAlign": "center",
              }
            }
          >
            <div
              className="ant-select-show-search ant-select-auto-complete ant-select ant-select-combobox ant-select-enabled"
              key="trigger"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onMouseDown={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "fontSize": "12px",
                  "width": "100%",
                }
              }
            >
              <div
                aria-autocomplete="list"
                aria-expanded={false}
                aria-haspopup="true"
                className="ant-select-selection
                          ant-select-selection--single"
                key="selection"
                role="combobox"
              >
                <div
                  className="ant-select-selection__rendered"
                >
                  <div
                    className="ant-select-selection__placeholder"
                    onClick={[Function]}
                    onMouseDown={[Function]}
                    style={
                      Object {
                        "WebkitUserSelect": "none",
                        "display": "block",
                        "userSelect": "none",
                      }
                    }
                    unselectable="on"
                  >
                    请输入名称
                  </div>
                  <ul>
                    <li
                      className="ant-select-search ant-select-search--inline"
                      key="__input"
                    >
                      <div
                        className="ant-select-search__field__wrap"
                      >
                        <input
                          className="ant-input ant-select-search__field"
                          disabled={false}
                          onChange={[Function]}
                          onKeyDown={[Function]}
                          type="text"
                          value=""
                        />
                        <span
                          className="ant-select-search__field__mirror"
                        >
                           
                        </span>
                      </div>
                    </li>
                  </ul>
                </div>
                <span
                  className="ant-select-arrow"
                  key="arrow"
                  onClick={[Function]}
                  style={
                    Object {
                      "WebkitUserSelect": "none",
                      "userSelect": "none",
                    }
                  }
                  unselectable="on"
                >
                  <b />
                </span>
              </div>
            </div>
          </td>
        </tr>
        <tr>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 5px",
                  "width": "90px",
                }
              }
            >
              <span>
                数量:
              </span>
              <span>
                <input
                  className="ant-input"
                  disabled={false}
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  style={
                    Object {
                      "fontSize": "12px",
                      "marginLeft": "5px",
                      "padding": "2px",
                      "width": "40px",
                    }
                  }
                  type="number"
                  value="1"
                />
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 0px",
                  "width": "90px",
                }
              }
            >
              <span>
                单位:
              </span>
              <span>
                <div
                  className="ant-select ant-select-enabled"
                  key="trigger"
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onMouseDown={[Function]}
                  onTouchStart={[Function]}
                  style={
                    Object {
                      "marginLeft": "5px",
                      "width": "55px",
                    }
                  }
                >
                  <div
                    aria-autocomplete="list"
                    aria-expanded={false}
                    aria-haspopup="true"
                    className="ant-select-selection
                              ant-select-selection--single"
                    key="selection"
                    onKeyDown={[Function]}
                    role="combobox"
                    tabIndex={0}
                  >
                    <div
                      className="ant-select-selection__rendered"
                    >
                      <div
                        className="ant-select-selection-selected-value"
                        key="value"
                        style={
                          Object {
                            "display": "block",
                            "opacity": 1,
                          }
                        }
                        title="副"
                      >
                        副
                      </div>
                    </div>
                    <span
                      className="ant-select-arrow"
                      key="arrow"
                      onClick={[Function]}
                      style={
                        Object {
                          "WebkitUserSelect": "none",
                          "userSelect": "none",
                        }
                      }
                      unselectable="on"
                    >
                      <b />
                    </span>
                  </div>
                </div>
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 0px",
                  "width": "90px",
                }
              }
            >
              <span>
                单价:
              </span>
              <span>
                <input
                  className="ant-input"
                  disabled={false}
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  style={
                    Object {
                      "fontSize": "12px",
                      "marginLeft": "5px",
                      "padding": "2px",
                      "width": "55px",
                    }
                  }
                  type="number"
                  value="undefined"
                />
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 0px",
                  "width": "90px",
                }
              }
            >
              <span>
                总价:
              </span>
              <span>
                <input
                  className="ant-input ant-input-disabled"
                  disabled={true}
                  onKeyDown={[Function]}
                  style={
                    Object {
                      "fontSize": "12px",
                      "marginLeft": "5px",
                      "padding": "2px",
                      "width": "55px",
                    }
                  }
                  type="text"
                  value=""
                />
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 0px",
                  "width": "80px",
                }
              }
            >
              <span>
                折扣:
              </span>
              <span>
                <input
                  className="ant-input"
                  disabled={false}
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  style={
                    Object {
                      "fontSize": "12px",
                      "marginLeft": "8px",
                      "padding": "2px",
                      "width": "40px",
                    }
                  }
                  type="number"
                  value="10"
                />
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "padding": "0 0 5px 0px",
                  "width": "110px",
                }
              }
            >
              <span>
                折扣后:
              </span>
              <span>
                <input
                  className="ant-input ant-input-disabled"
                  disabled={true}
                  onKeyDown={[Function]}
                  style={
                    Object {
                      "fontSize": "12px",
                      "marginLeft": "8px",
                      "padding": "2px",
                      "width": "60px",
                    }
                  }
                  type="text"
                  value=""
                />
              </span>
            </div>
          </td>
          <td>
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "marginLeft": "30px",
                  "padding": "0 0 5px 0px",
                  "width": "50px",
                }
              }
            >
              <a
                href=""
                onClick={[Function]}
                style={
                  Object {
                    "marginRight": "12px",
                  }
                }
              >
                添加
              </a>
              <br />
              <span
                key="trigger"
                onClick={[Function]}
                onMouseDown={[Function]}
                onTouchStart={[Function]}
              />
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    style={
      Object {
        "alignItems": "flex-end",
        "display": "flex",
        "justifyContent": "space-around",
        "marginTop": "15px",
      }
    }
  >
    <div>
      <span>
        原总价:
      </span>
      <span>
        <input
          className="ant-input ant-input-disabled"
          disabled={true}
          onKeyDown={[Function]}
          style={
            Object {
              "marginLeft": "12px",
              "width": "80px",
            }
          }
          type="text"
          value={0}
        />
      </span>
    </div>
    <div>
      <span>
        折扣后:
      </span>
      <span>
        <input
          className="ant-input ant-input-disabled"
          disabled={true}
          onKeyDown={[Function]}
          style={
            Object {
              "marginLeft": "12px",
              "width": "80px",
            }
          }
          type="text"
          value={0}
        />
      </span>
    </div>
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
        }
      }
    >
      <span>
        实收:
      </span>
      <span>
        <input
          className="ant-input"
          disabled={false}
          onChange={[Function]}
          onKeyDown={[Function]}
          style={
            Object {
              "fontSize": 20,
              "height": "40px",
              "marginLeft": "12px",
              "width": "120px",
            }
          }
          type="number"
          value="0"
        />
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "space-between",
        "marginTop": "15px",
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
        }
      }
    >
      <div>
        备注:
      </div>
      <div>
        <textarea
          className="ant-input"
          onChange={[Function]}
          onKeyDown={[Function]}
          rows={2}
          style={
            Object {
              "marginLeft": "25px",
              "resize": "none",
              "width": "395px",
            }
          }
          value=""
        />
      </div>
    </div>
    <div>
      <button
        className="ant-btn ant-btn-primary"
        onClick={[Function]}
        type="button"
      >
        <span
          key=".0"
        >
          再次购买
        </span>
      </button>
    </div>
    <div />
  </div>
</div>
`;
